src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -276,18 +253,14 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
       // verify that file is valid
       if (!this.fileValidity) {
         // NB: as this is validity according to the component, do not overwrite current error message
-        this.snackbarText = 'Invalid file. It will not be saved.'
-        this.snackbar = true
         return // don't add to list
       }
 
       // verify that file doesn't already exist
       if (this.authorization.files.find(f => f.file.name === file.name)) {
         // put file uploader into manual error mode by setting custom error message
-        this.customErrorMessage[index] = 'Duplicate file.'
-
-        this.snackbarText = 'File already uploaded. It will not be saved again.'
-        this.snackbar = true
+        this.customErrorMessage[index] = 'Duplicate file'
+        this.$forceUpdate() // force file upload component to react
```

Comment:
@JazzarKarim FYI! Without this, the file upload component did not react to the new custom error message (probably because it's a changed array element instead of an updated variable). Previously, the code to update the snackbar somehow triggered a DOM update. Now, I need to do this (or refactor a lot) so the custome error message displays.

Pass this on to any other UI developers that should know this ;)

Reply:
First time I've seen something like this. Good to know!

I'm looking at it now, I saw this:
![forceupdate](https://github.com/bcgov/business-create-ui/assets/122301442/3fc90013-87cf-47be-9159-5120ccb0db48)

So this forces the entire thing to re-render?

Reply:
> So this forces the entire thing to re-render?

Unclear. I see what the docs say, but I'm calling this on the component instance (VM) not the app instance, and I don't see a flicker or anything. It might be smart and only re-render what needs to be updated...?

Reply:
That's exactly what I'm wondering about too. If that's the case, then it's awesome. I'm assuming it's that since if it is indeed re-rendering the entire thing, it'll flicker or there'll be hiccup as you mentioned. Anyway, was just wondering.

Reply:
Try it for yourself and let me know if you notice anything: https://business-create-dev--pr-689-59a0htqn.web.app/continuation-in-business-home?id=TZDikc3p6I

1. upload a valid PDF
2. try to upload it again
3. see error

Reply:
Yup, it worked perfectly with no hiccups. 
![duplicate](https://github.com/bcgov/business-create-ui/assets/122301442/8fb51c6f-5f34-4f07-b2d5-6c8419c6fcb3)

Sounds good then, thanks Sev! This is actually super useful, will probably use it sometime down the line if I'm having reactivity issues.

Reply:
This is the first time I've needed it, probably due to deep prop reactivity not working.